test/channels.jl

```diff
@@ -553,6 +553,11 @@ let a = []
     sleep(0.2)
     @test a == [1]
 end
+let a = []
+    Timer(t -> push!(a, 1), 0.01, interval = 0, spawn = true)
+    sleep(0.2)
+    @test a == [1]
```

Comment:
Minor suggestion, for these kinds of tests depending on asynchronous behaviour I like to use `timedwait()`:
```suggestion
    @test timedwait(() -> a == [1], 10) === :ok
```

The advantage is that you can give it a long timeout to handle slow CI machines while still processing the change relatively quickly after it happens (default poll interval is 0.1s).

Reply:
Thanks!